components/schemas/scoped-variables/URLSource.yml

```diff
@@ -14,7 +14,15 @@ properties:
     type: object
     required:
       - url
+      - headers
```

Comment:
auth_token_url is not `omitempty`, and therefore must be marked as required.

![image](https://github.com/cycleplatform/api-spec/assets/5110855/e015dcec-d6aa-46ae-ae67-af714017b48f)